### PR TITLE
Remove unnecessary awaits for updateComplete

### PR DESCRIPTION
**Pull request by dlockhart**
Now that everything is using the new `fixture` that automatically `await`s nested Lit components' `updateComplete`s, all of our tests that were doing that manually can... not do that.

### CODE REVIEW

File: components/tooltip/test/tooltip.test.js [hunk 1]
```diff
@@ -185,11 +185,7 @@ describe('d2l-tooltip', () => {
 				await oneEvent(tooltipFixture, 'd2l-tooltip-show');
 				await aTimeout();
 
-				const eventObj = document.createEvent('Events');
-				eventObj.initEvent('keydown', true, true);
-				eventObj.keyCode = 27;
-
-				setTimeout(() => document.dispatchEvent(eventObj));
+				setTimeout(() => sendKeys('press', 'Escape'));
```
Comment (dlockhart) [inline components/tooltip/test/tooltip.test.js:188]: Just a little fixy-fixy while I was in here...

File: components/list/test/list.test.js [hunk 2]
```diff
@@ -6,15 +6,6 @@ import '../list-item-content.js';
 import { expect, fixture, focusElem, html, oneEvent, sendKeysElem, waitUntil } from '@brightspace-ui/testing';
 import { runConstructor } from '../../../tools/constructor-test-helper.js';
 
-const awaitListElementUpdates = async(rootElement, queries) => {
```
Comment (dbatiste) [inline components/list/test/list.test.js:9]: Super gross. 🤮  Nice to see this gone!

File: components/selection/test/selection.test.js [hunk 3]
```diff
@@ -123,7 +123,6 @@ describe('d2l-selection-input', () => {
 				<d2l-selection-input key="key3" label="label3" disabled></d2l-selection-input>
 			</d2l-test-selection>
 		`);
-		await el.updateComplete;
 		await nextFrame();
```
Comment (dbatiste) [inline components/selection/test/selection.test.js:126]: I wonder if this `nextFrame()` is still needed? I'm imagining you already tried to remove it...?

Comment (dlockhart) [reply components/selection/test/selection.test.js:126]: There's a lot of these in here so I didn't want to mess with too much that I wasn't completely confident would be covered by the new `fixture`.

**Review by dbatiste — APPROVED** [review]

**Pull request merged**